### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by limdauto**

### CODE REVIEW

**Comment by lorenabalan** [comment]
Whiiiile you're here, mind if you update the import in https://github.com/quantumblacklabs/kedro-starters/blob/4b37476d6296f5031cf89b0e86f0149d26822e66/spaceflights/%7B%7B%20cookiecutter.repo_name%20%7D%7D/docs/source/conf.py#L50 please? :D :D :D

File: spaceflights/{{ cookiecutter.repo_name }}/.gitignore [hunk 1]
```diff
@@ -20,7 +20,7 @@ logs/**
 !.gitkeep
 
 # keep also the example dataset
-!data/01_raw/iris.csv
+!data/01_raw/*.csv
```
Comment (MerelTheisenQB) [inline spaceflights/{{ cookiecutter.repo_name }}/.gitignore:23]: Spaceflights also has an excel file right?

Comment (limdauto) [reply spaceflights/{{ cookiecutter.repo_name }}/.gitignore]: Thanks. Updated. Also updated the import @lorenabalan

**Review by lorenabalan — APPROVED** [review]
💯

**Review by DmitriiDeriabinQB — APPROVED** [review]
🥇

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Pull request merged**